frontend/src/component/changeRequest/ChangeRequest/Changes/Change/EditChange.tsx

```diff
@@ -208,7 +209,7 @@ export const formatUpdateStrategyApiCode = (
     }
 
     const url = `${unleashUrl}/api/admin/projects/${projectId}/change-requests/${changeRequestId}/changes/${changeId}`;
-    const payload = JSON.stringify(strategy, undefined, 2);
+    const payload = JSON.stringify(strategy, apiPayloadConstraintReplacer, 2);
```

Comment:
Format the payload of edit change consistently with how we did it in https://github.com/Unleash/unleash/pull/10175